zds/searchv2/models.py

```diff
@@ -6,73 +6,69 @@
 from django.db import models
 from django.conf import settings
 
-from elasticsearch.helpers import parallel_bulk
-from elasticsearch import ConnectionError
-from elasticsearch_dsl import Mapping
-from elasticsearch_dsl.query import MatchAll
-from elasticsearch_dsl.connections import connections
-
 from django.db import transaction
+from typesense import Client
```

Comment:
```suggestion
from typesense import Client as SearchEngineClient
```
pour que ce soit plus simple par la suite de comprendre de quel client on parle.